package.json

```diff
@@ -77,6 +77,7 @@
     "build": "npm run clean && npm run build:cjs && npm run build:es && npm run build:umd && npm run build:amd && npm run copy",
     "preversion": "npm run build && git push",
     "postversion": "git push && git push --tags",
+    "pretest": "enzyme-adapter-react-install 16",
```

Comment:
In Travis `enzyme-adapter-react-install` will use `process.env.REACT` instead `16`

Reply:
Also for local tests can be used:
`REACT=0.14 npm test`